docs/migration-tutorial.md

```
@@ -0,0 +1,587 @@
+  --source-context "${SOURCE_CONTEXT}" \
+  --destination-context "${TARGET_CONTEXT}" \
+  --pvc-name mongodb-data \
+  --pvc-namespace "${NAMESPACE}" \
+  --endpoint nginx-ingress \
+  --subdomain "${NODE_IP}.nip.io" \
+  --ingress-class nginx \
+  --verify
+```
+
+> **Minikube setup:** Both minikube clusters must share the same Docker/Podman network for the InternalIP to be reachable. Use `make setup-minikube-same-network` from this repo, or create clusters with `--network=minikube`. The nip.io wildcard DNS service resolves `<IP>.nip.io` back to the IP, providing a valid hostname for the Ingress without any DNS setup.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)